AUTHORS.md

```diff
@@ -8,10 +8,10 @@ Manoa.
 The following people have contributed code to the project (alphabetical by last name)
```

Comment:
```suggestion
The following people have contributed code to the project (alphabetical by name)
```

As per our [Code of Conduct](https://github.com/GenericMappingTools/pygmt/blob/v0.2.1/CODE_OF_CONDUCT.md) to be inclusive, I think we should just sort alphabetically by name (i.e. full name) for these reasons:

1. Some people don't have family names (see e.g. [Indonesian names](https://en.wikipedia.org/wiki/Indonesian_names#Indonesian_naming_traditions)). Also, 'last' name isn't applicable to every culture. My last name is "Leong" for example, but it's technically positioned "first" from where I grew up, and we've always just used "name" rather than "first name" "last name".
2. People who get married and change their last name will keep their alphabetical position in the list (unless their last name is positioned first :upside_down_face:).

Reply:
Great point @weiji14. Let's do it.